superset-frontend/cypress-base/cypress.config.ts

```diff
@@ -37,30 +36,10 @@ export default eyesPlugin(
       openMode: 0,
     },
     e2e: {
+      experimentalMemoryManagement: true,
```

Comment:
I see we are doing `{ testIsolation: false }` in the describe blocks, is there a reason to not just add that to the config instead?

Reply:
It's due to [default test isolation](https://docs.cypress.io/guides/references/migration-guide#Test-Isolation) in Cypress 12. While there are many tests sharing context, as evidenced by the amount of `{ testIsolation: false }`, the remaining are isolated and should be encouraged instead.

We'll probably need to refactor these context-sharing tests later to be isolated in future PRs.